src/inmanta/deploy/scheduler.py

```diff
@@ -921,7 +927,9 @@ async def _new_version(
             self._timer_manager.remove_timers(deleted)
             # Install timers for initial up-to-date resources. They are up-to-date now,
             # but we want to make sure we periodically repair them.
-            self._timer_manager.update_timers(up_to_date_resources | (transitive_unblocked - self._state.dirty))
+            self._timer_manager.update_timers(
+                (last_deploy_time.keys() | transitive_unblocked | resources_with_reset_requires) - self._state.dirty
```

Comment:
I think `last_deploy_time.keys() - self._state.dirty` is equivalent to `up_to_date_resources`, but I have no objections. Except that if we do it like this, we should probably document above that either both should be set or both should be None.